### PR TITLE
Integration of Evmos

### PR DESCRIPTION
**Pull request by salvatore-galaxystaking**
#https://github.com/osmosis-labs/assetlists/pull/115

### CODE REVIEW

**Comment by jonator** [comment]
@salvatore-galaxystaking plans/timelines for creating the a pool and adding it to the swap manager?

**Comment by salvatore-galaxystaking** [comment]
I just want to help and do some work.
Following the telegram group there are some people who are willing to set up a pool but are waiting for frontend integration.

**Comment by ValarDragon** [comment]
A lot more has to be done for evmos atm. They use different key types, so the default logic is unsafe.

**Comment by fedekunze** [comment]
@ValarDragon please wait until the planned upgrade today, the transfer to the same keys (secp256k1 -> secp256k1) should be fixed then

**Comment by fedekunze** [comment]
@salvatore-galaxystaking please use the following SVG
evmos-icon

**Comment by jonator** [comment]
Could someone please add me to the telegram? My user: `@jator0`.

**Comment by fedekunze** [comment]
@salvatore-galaxystaking forgot the SVG

Evmos-Icon

File: public/assets/tokens/evmos.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="940" height="660" viewBox="0 0 940 660" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jonator) [inline public/assets/tokens/evmos.svg:1]: Please sync default appearance of SVG to match PNG

**Review by jonator — CHANGES_REQUESTED** [review]

File: public/assets/tokens/evmos.svg (hunk 1 above)
Comment (salvatore-galaxystaking) [reply public/assets/tokens/evmos.svg]: synced

**Comment by jonator** [comment]
@salvatore-galaxystaking It appears the PNG is rotated on the assets page.. please use the SVG instead. In fact, I elect to remove the PNG entirely.

**Comment by JeremyParish69** [comment]
Looks like EVMOS uses CosmosSDK 0.45.1 and ibc-go, and therefore should include feature `'ibc-go'` on config.ts
also, could include high, avg, and low gas step on config.ts?

**Comment by salvatore-galaxystaking** [comment]
> Looks like EVMOS uses CosmosSDK 0.45.1 and ibc-go, and therefore should include feature `'ibc-go'` on config.ts also, could include high, avg, and low gas step on config.ts?

done - please highlight which values for gas step may suit best

**Comment by JeremyParish69** [comment]
> done - please highlight which values for gas step may suit best

Thank you. Without having much experience with EVMOS, myself, I can't speculate as to which gas rates would be appropriate. I can only relay the gist as part of the request: that Low rate should be a savings, but more likely to fail; and that High rate should be much more likely to succeed and quickly, even when busy; and that Avg rate should still more the vast majority of the time.

Eveything else looks good to me.

**Review by jonator — APPROVED** [review]

File: src/config.ts [hunk 2]
```diff
@@ -2423,4 +2429,46 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx'],
 		explorerUrlToTx: 'https://explorer.decentr.net/transactions/{txHash}?networkId=mainnet',
 	},
+	{
+		rpc: 'https://rpc-evmos.keplr.app/',
+		rest: 'https://lcd-evmos.keplr.app/',
+		chainId: 'evmos_9001-1',
```
Comment (JeremyParish69) [inline src/config.ts:2435]: update chain-id to -2

Comment (JeremyParish69) [inline src/config.ts:2435]: update chain-id to -2

File: src/config.ts [hunk 3]
```diff
@@ -922,6 +922,12 @@ export const IBCAssetInfos: {
 		destChannelId: 'channel-1',
 		coinMinimalDenom: 'udec',
 	},
+	{
+		counterpartyChainId: 'evmos_9001-1',
```
Comment (JeremyParish69) [inline src/config.ts:926]: update chain-id to -2

Comment (JeremyParish69) [inline src/config.ts:926]: update chain-id to -2

**Comment by salvatore-galaxystaking** [comment]
@JeremyParish69 updated to evmos_9001-2

**Comment by JeremyParish69** [comment]
This should be trying to merge into Frontier branch, not master.
Also, the svg does not match the images in the Cosmos Chain Registry; which version is correct?

**Comment by JeremyParish69** [comment]
This should be trying to merge into Frontier branch, not master.
Also, the svg does not match the images in the Cosmos Chain Registry; which version is correct?

**Pull request closed without merging**